docs/source/python/data.rst

```diff
@@ -431,4 +431,44 @@ around, so if your data is already in table form, then use
 Custom Schema and Field Metadata
 --------------------------------
 
-TODO
+The metadata used for describing the data in the schema or a field can also
+be added with the use of ``replace_schema_metadata`` or ``with_metadata``.
+
+To customize the metadata of the existing table you can use
+``replace_schema_metadata``:
+
+.. ipython:: python
+
+   table.schema.metadata # empty
+   table = table.replace_schema_metadata({"f0": "First dose"})
+   table.schema.metadata
+
+To customize the metadata of the field from the table schema you can use
+``with_metadata``:
+
+.. ipython:: python
+
+   field_f1 = table.schema.field("f1")
+   field_f1.metadata # empty
+   field_f1 = field_f1.with_metadata({"f1": "Second dose"})
+   field_f1.metadata
+
+Both options create a shallow copy of the data and do not in fact change
+Schema which is immutable. To change the metadata in the schema of the table we
```

Comment:
The `To change the metadata in the schema of the table we created a new object`, does this refer to the `replace_schema_metadata` example above?

Reply:
Yes, I guess it is not clear - will reword to be more explicit.